whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/OrgSpec.groovy

```diff
@@ -3652,4 +3640,48 @@ class OrgSpec extends BaseQueryUpdateSpec {
                 [ "Country not recognised: FF" ]
     }
 
+    def "modify organisation, org-type:OTHER, change country code with user password"() {
```

Comment:
Is there also a test for create organisation with country code? This should always be allowed.